### PR TITLE
Fix handling of EM_FROZEN_CACHE=0. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
Split out from #17662

File: .circleci/config.yml [hunk 1]
```diff
@@ -209,7 +209,21 @@ commands:
...
+            # that are not included by "./embuilder build ALL".  For example the
+            # PIC version of libSDL which is used by test_sdl2_misc_main_module
+            EM_FOO: "0"
+            FOO: "0"
+            EM_BAR: "1"
+            BAR: "1"
+            EM_FROZEN: "0"
+            FROZEN: "0"
+            EM_XX: "0"
+            FROZEN: "0"
```
Comment (kripken) [inline .circleci/config.yml:222]: What are all these env vars?

Comment (sbc100) [reply .circleci/config.yml:222]: Sorry just debugging .. seeing some very odd behavior on circle ci where it is seemingly ignoring some env vars.

**Comment by bbb11223344556677** [comment]
why anyone people wasting their time with this? everything reported is FAKE and the real information will be posted shortly

**Comment by sbc100** [comment]
This change ended up landing as part of #17659

**Pull request closed without merging**